pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021-2023 Tigera, Inc. All rights reserved.
```

Comment:
There are no changes to this file it seems.

Reply:
Yeah, but the CI was complaining about it so I had to change it. I'll try reverting...

Reply:
I think I just need to leave it, shouldn't be any harm. The CI and pre-commit hooks won't let me adjust this.